docs/extensions/netsuite/considerations.md

```diff
@@ -28,6 +28,14 @@ Request payload example:
 }
 ```
 
+### CURL version example
```

Comment:
The title might potentially be a bit unnecessary. Consider following up with a simple sentence describing that "you can execute this change via the following command…"